scripts/labeled_stream.py

```diff
@@ -0,0 +1,114 @@
+"""
```

Comment:
remove this script

Reply:
Can't remove it from this branch, will remove it from master